packages/crypto/src/math.ts

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Should we add some tests here since we're exposing these fxns? Also nit: consistent docstrings for exported fxns

Reply:
We're exposing these functions to use in `crypto.ts`; not quite exporting to the outside world (yet). But you're right, we should probably beef up test coverage in this library overall

Reply:
Ahh mb but yeah haha wouldn't hurt

Reply:
I'm gonna punt on adding tests if that's okay — will leave that as an open issue